workflow/rocoto/workflow_tasks.py

```diff
@@ -1133,28 +1133,28 @@ def _get_ecengroups():
 
     def esfc(self):
 
-        self._is_this_a_gdas_task(self.cdump, 'esfc')
+        #self._is_this_a_gdas_task(self.cdump, 'esfc')
```

Comment:
More dead code. Not going to list them all.

Reply:
Yes, just keeping them as ref and will delete them all later